applications/PfemFluidDynamicsApplication/tests/cut_pfem_tests/dam_break_2d/ProjectParameters.json

```diff
@@ -0,0 +1,152 @@
+{
+    "problem_data" : {
+        "problem_name"   : "dam_break_2d",
+        "start_time"     : 0.0,
+        "end_time"       : 0.006,
+        // "end_time"       : 0.3,
```

Comment:
comments in json may be dangerous in the master

Reply:
Why? First time I heard about this.

Reply:
@roigcarlo told me :) when running tests not all interpreters catch it properly

Reply:
Yep, please no comments in the jsons if possible. Our parser "should" accept it, but we break compatibility with basically any other json reader.